samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazor/Startup.cs

```diff
@@ -39,7 +39,8 @@ public void ConfigureServices(IServiceCollection services)
             
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
 
-            services.AddPredictionEnginePool<ModelInput, ModelOutput>().FromFile(_modelPath);
+            services.AddPredictionEnginePool<ModelInput, ModelOutput>()
+                    .FromFile(_modelPath);
```

Comment:
This change has broken the docs, the line was referenced by its number.
I've opened https://github.com/dotnet/docs/pull/14821 to fix that in the docs repo.